bread/layout/base.py

```diff
@@ -116,7 +116,8 @@ def resolve(self, context):
         parts = self.fieldname.split(".")
         # test if the value has a matching get_FIELDNAME_display function
         value = hg.resolve_lookup(
-            object, f"{'.'.join(parts[:-1])}.get_{parts[-1]}_display"
+            object,
+            f"{''.join([part + '.' for part in parts[:-1]])}get_{parts[-1]}_display",
```

Comment:
Can you quickly explain what this change improves/fixes?

Reply:
I had explained it in the commit message ;)
if there is only one part we ended up with an unwanted leading dot (e.g. ".get_FIELDNAME_display")

Reply:
I see, missed the commit message. This could also be solved with
```python
f"{'.'.join(parts[:-1])}.get_{parts[-1]}_display".lstrip(".")
```
which I slightly prefer, but it is fine either way.